### PR TITLE
Add a metric to Hook for webhook status reports

### PR DESCRIPTION
**Pull request by ashleydavies**
On our deployment, we'd like to have some more metrics to keep an eye on Hook's operations, and we would like to notice issues like increased traffic which might need to be looked into.

Thought this may be a change you are interested in integrating as I imagine this may be quiet useful to others too.

cc @petemounce @DoomGerbil @majolo @lottec @efarem

### CODE REVIEW

File: prow/hook/server.go [hunk 1]
```diff
@@ -20,14 +20,14 @@ import (
...
 	"fmt"
+	"github.com/sirupsen/logrus"
 	"io/ioutil"
 	"net/http"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/sirupsen/logrus"
```
Comment (kargakis) [inline prow/hook/server.go]: Please revert this change

**Comment by kargakis** [comment]
lgtm

The diff is small so once you rebase on top of https://github.com/kubernetes/test-infra/pull/8933, squash to a single commit and I will tag

**Comment by ashleydavies** [comment]
Thanks again @kargakis! Just to let you know I've rebased and squashed the changes
